packages/mux-player/src/html.ts

```diff
@@ -108,24 +108,29 @@ export function processPart(part: Part, value: unknown): void {
     processPropertyIdentity(part, value);
 }
 
-const templates = new WeakMap<TemplateStringsArray, HTMLTemplateElement>();
+const templates = new Map<string, HTMLTemplateElement>();
```

Comment:
question(blocking): Does this effectively mean no `HTMLTemplateElement` will ever be gc'ed? I'm not seeing any mechanism to do a `templates.delete(stringKey)`

Reply:
yes, that's right. I don't really see an other way to do it at the moment. 
it should not add more and more templates though, it only goes until each tagged template is evaluated.

quickly tested, it's about 5 entries changing all permutations and such

Reply:
Cool. I think this would be resolved by my proposal of the `<mux-player/>` instance "owning" the reference to the template instead of the cache `Map` object, but that would be a bigger refactor of the underlying architecture. I think something in that direction makes more sense longer term, but let's treat that as a separate ticket.